ca/certificates.go

```diff
@@ -245,31 +244,53 @@ func (rca *RootCA) ParseValidateAndSignCSR(csrBytes []byte, cn, ou, org string)
 		return nil, err
 	}
 
-	return cert, nil
+	// Append the first root CA Cert to the certificate, to create a valid chain
+	// Get the first Root CA Cert on the bundle
+	firstRootCA, _, err := helpers.ParseOneCertificateFromPEM(rca.Cert)
+	if err != nil {
+		return nil, err
+	}
+	// Convert the first root CA back to PEM
+	firstRootCAPEM := helpers.EncodeCertificatePEM(firstRootCA[0])
```

Comment:
It looks like `ParseOneCertificateFromPEM` can return a `nil` first return value even when it doesn't return an error:

https://github.com/cloudflare/cfssl/blob/master/helpers/helpers.go#L290

Do we need to check for this?